bundle/metadata/dependencies.yaml

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: MIT
 # SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
```

Comment:
Double copyright headers. I don't think this file is auto-generated, so it is worth checking it doesn't re-add the headers each time the bundle is generated.